### ISSUE TITLE
Provide Windows implementation for function ImFileExist

### ISSUE THREAD

**Issue by aandrejevas** [ocornut/imgui_test_engine#22]
![image](https://user-images.githubusercontent.com/53169542/233173817-53bd52ce-8f04-4d6d-b674-32eaeb9f8160.png)
Hello, right now this function shown in the picture above does not compile on Windows. An implementation should be provided that would compile on Windows. This function (ImFileExist) is defined in file imgui_te_utils.cpp.

**Comment by ocornut** [comment]
Please be reasonable and explain what doesn’t compile for you and provide details and context and compiler info, because clearly it compiles for us seeing we are building the test engine everyday.

**Comment by aandrejevas** [comment]
Yes, sorry, I made many assumptions. I just saw the function stat being used and I instantly thought that this was an implementation not for Windows. Now I know that Microsoft has implemented many POSIX functions as extensions also.

The minimal demo app compiles and runs fine using Visual Studio.

I am trying to compile my app with gcc. These are the errors I get (the command in the picture is not the one I actually use):
![image](https://user-images.githubusercontent.com/53169542/233229164-8c54faeb-dec6-42f6-b4d2-e48fff4c1a67.png)

This is the info about the compiler I use:
![image](https://user-images.githubusercontent.com/53169542/233230217-8a97832c-828b-4eaa-8116-33f9b99b1434.png)

So now I think that something with the includes in the file imgui_te_utils.cpp might be wrong. I added the includes that I read in a microsoft website are required by the stat function. After that everything compiled and I was able to run my app. These are my changes:
![image](https://user-images.githubusercontent.com/53169542/233232035-b369cd1b-10da-4b9c-97b3-138e95c38482.png)
